.github/workflows/gradle.yml

```diff
@@ -65,6 +65,12 @@ jobs:
         with:
           python-version: '3.7'
 
+      - name: Format & Generate
```

Comment:
can we have a task called generate instead which depends on generateSeed generateProtocolClassFiles generateJsonSchema2Pojo instead of expending a list?

Or maybe we can have `format` to depend on everything that generates?

Reply:
Sounds good. Added a `generate` step for `generateSeed`, `generateProtocolClassFiles`, and `generateJsonSchema2Pojo` that `format` depends on.